docs/contributing/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+## How to contribute
+
+Pocket is an open source project, and as such we welcome any contribution from anyone on the internet.
+
+Please fork, code and submit a Pull Request for the team to review and merge. We ask that you please follow the
+guidelines below in order to submit your contributions for review:
+
+### High impact or architectural changes
+
+Reach out to us on [Telegram](https://t.me/POKTnetwork) and start a discussion with the team regarding your change
+before you start working. Communication is key for open source projects and asynchronous contributions.
+
+For an active research forum, checkout and post on [our forum](https://research.pokt.network).
+
+### Coding style
+
+- Code must adhere to the official Go formatting guidelines (i.e. uses [gofmt](https://golang.org/cmd/gofmt)).
+- (Optional) Use [Editor Config](https://editorconfig.org) to help your Text Editor keep the same formatting used
+  throughout the project.
+- Code must be documented adhering to the official Go commentary guidelines.
+- Pull requests need to be based on and opened against the `main` branch.
+
+### Criteria for PR Evaluation
+
+- Quality: the PR should include instructions on how to test the submitted code. Automated tests are not a must in all
+  situations, but always strongly encouraged
+- Criteria expressed in [this document](TASTE.md)
+- Context: make it easy for the reviewers to understand what is being addressed by the PR and why it's important to the
+  project
```

Comment:
DITTO comment above around "breakpoints" being in different places.

Reply:
Indeed; I will ignore for the moment since it's meaningless whitespace in markdown, but will include in the next PR.